yjit/src/stats.rs

```diff
@@ -198,9 +198,10 @@ macro_rules! make_counters {
 
 /// The list of counters that are available without --yjit-stats.
 /// They are incremented only by `incr_counter!` and don't use `gen_counter_incr`.
-pub const DEFAULT_COUNTERS: [Counter; 7] = [
+pub const DEFAULT_COUNTERS: [Counter; 8] = [
     Counter::code_gc_count,
     Counter::compiled_iseq_entry,
+    Counter::iseq_entry_cold,
```

Comment:
This is kind of the opposite of `compiled_iseq_entry`, so maybe it's useful to call it `cold_iseq_entry` to contrast them.

Reply:
Good idea